Tests/Tests/SDWebImageTestCache.m

```diff
@@ -98,14 +121,29 @@ - (void)setData:(nullable NSData *)data forKey:(nonnull NSString *)key {
 }
 
 - (NSUInteger)totalCount {
-    return [self.fileManager contentsOfDirectoryAtPath:self.cachePath error:nil].count;
+    NSUInteger count = 0;
+    @autoreleasepool {
+        count = [self.fileManager contentsOfDirectoryAtPath:self.cachePath error:nil].count;
```

Comment:
Do we need to change this into `contentsOfDirectoryAtURL` ?

Reply:
Or any other faster way to calculate `file number under the directory`

Reply:
Oh, this is the test utils, don't mind

Reply:
There are no ``contentsOfDirectoryAtURL:`` like urls.

``subpathsAtPath:`` can do, also u can use ``enumeratorAtURL:`` and with enumerator call count, just like 

```oc
// SDDiskCache.m

- (NSUInteger)totalCount {
    NSUInteger count = 0;
    @autoreleasepool {
        NSURL *diskCacheURL = [NSURL fileURLWithPath:self.diskCachePath isDirectory:YES];
        NSDirectoryEnumerator<NSURL *> *fileEnumerator = [self.fileManager enumeratorAtURL:diskCacheURL includingPropertiesForKeys:@[] options:(NSDirectoryEnumerationOptions)0 errorHandler:nil];
        count = fileEnumerator.allObjects.count;
    }
    return count;
}
```